### ISSUE TITLE
Right click menu "Terrain" action not functional, reports error in debug console

### ISSUE THREAD

**Issue by dscp46** [resiliencetheatre/rpi4edgemap#3]
Hi,

On the latest build of rpi4edgemap web UI, the "Terrain" action in the right click menu is not properly set up.
However, it seems to be properly set up if you're invoking the same action from the pancake menu (see attached video for reference).

Here is a download link to a video that describes the issue: [https://cloud.luthienstar.fr/index.php/s/m8Bz9qopf4bN532](https://cloud.luthienstar.fr/index.php/s/m8Bz9qopf4bN532)

Kind regards,

**Comment by resiliencetheatre** [comment]
Agree. I am still evaluating is that right click menu usable at all. I have not found a way to activate it from mobile browser (namely Safari) - any idea how to do that? Therefore that menu is still work in progress.

**Comment by resiliencetheatre** [comment]
Fixed upstream.
